include/vkgcDefs.h

```diff
@@ -619,6 +620,7 @@ struct PipelineShaderOptions {
   /// disables limiting the number of thread-groups to launch. This field is ignored for graphics shaders.
   unsigned maxThreadGroupsPerComputeUnit;
 
+  unsigned subgroupSize;       ///< Control subgroupSize
```

Comment:
You should comment that 0 is a special value, and say what it means.

Reply:
A better comment might be

```
///< The number of threads in the subgroup. 0 means the size is unspecified, and any value can be chosen.
```

I don't know which component will actually choose the value.  You can add that to the comment if you want.

Reply:
Thanks, I will update it.